rs/messaging/src/routing/stream_handler.rs

```diff
@@ -1096,7 +1102,10 @@ fn generate_reject_response_for(reason: RejectReason, request: &Request) -> Requ
     let (code, message) = match reason {
         RejectReason::CanisterMigrating => (
             RejectCode::SysTransient,
-            format!("Canister {} is migrating", request.receiver),
+            format!(
+                "Canister {} or {} is migrating",
+                request.receiver, request.sender
+            ),
```

Comment:
I'd stick with a plain `"Canister migration in progress"` here. It must obviously be one of `receiver` or `sender`. We can later go back to a more precise error message if we break this down into multiple reject signals, depending on context.